core/src/test/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManagerTests.java

```diff
@@ -39,13 +39,13 @@
 public class DelegatingReactiveAuthenticationManagerTests {
 
 	@Mock
-	ReactiveAuthenticationManager delegate1;
+	private ReactiveAuthenticationManager delegate1;
```

Comment:
Please leave this cleanup out for now or place it in a polish commit. That's so that the commit that adds the feature only has the necessary changes to add that feature.

Reply:
Done.